gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -17916,6 +18056,9 @@ properties:
       Omentum:
         $ref:
           - "_terms_enum.yaml#/omentum/common"
+      Uterus:
+        $ref:
+          - "_terms_enum.yaml#/uterus_nos/common"
```

Comment:
does this need to be uterus instead of uterus_nos?

Reply:
In _terms_enums.yaml, the term is "uterus_nos" and the "termDef" describes the term as "Uterus",  so I think this is okay.